### DISCUSSION TITLE
Error upgrading rehype-stringify 9.0.4 to 10.0.0.

### DISCUSSION THREAD

**Discussion by bfdes** [unifiedjs/unified#231] [Q&A] [answered]
I'm getting a `TypeError` saying I cannot call `process` without supplying a `Compiler` after upgrading rehype-stringify. I find this confusing because rehype-stringify _is_ the AST to string compiler in my unified pipeline.

Here is the PR at attempts to perform the upgrade: https://github.com/bfdes/bfdes.in/pull/78. You can see that the tests which invoke the pipeline fail. 

Here is a snippet of the pipeline, just in case the cause of the error is obvious from the snippet alone:

```javascript 
import rehypeKatex from "rehype-katex";
import rehypeStringify from "rehype-stringify";
import remarkGemoji from "remark-gemoji";
import remarkGFM from "remark-gfm";
import remarkMath from "remark-math";
import remarkParse from "remark-parse";
import remarkRehype from "remark-rehype";
import { unified } from "unified";
import { VFile } from "vfile";
import { matter } from "vfile-matter";

const vfile = new VFile("some string");
matter(vfile, { strip: true });

await unified()
  .use(remarkParse)
  .use(remarkGemoji)
  .use(remarkGFM)
  .use(remarkMath)
  .use(remarkRehype)
  .use(rehypeKatex)
  .use(rehypeStringify)
  .process(vfile)
  .then((vfile) => vfile.toString());
```

I can't see any API changes documented in the release notes to say I should change the way I invoke rehype-stringify.

**Comment by wooorm** [comment, marked as answer]
Update all these packages, not some. Particularly unified, which is likely the root cause

**Comment by bfdes** [comment]
Thanks. Upgrading all the unified packages at once got rid of the error.
